Program.cs

```diff
@@ -30,7 +30,7 @@ public static void Main(string[] args)
             builder.Services.AddSingleton<FileStorageService>();
             builder.Services.AddScoped<AuthService>();
             builder.Services.AddScoped<ProjectLikeService>();
-            builder.Services.AddDbContext<MainAppContext>(options => options.UseSqlServer(conf.GetConnectionString("Default")));
```

Comment:
Please, use Default for connection string instead of DefaultConnection, as all other contributors are using the Default.

Reply:
ok, my bad .